frontend/src/metabase/components/ItemsTable/BaseItemsTable.styled.tsx

```diff
@@ -2,21 +2,32 @@
 import { css } from "@emotion/react";
 // eslint-disable-next-line no-restricted-imports
 import styled from "@emotion/styled";
-import type { HTMLAttributes } from "react";
+import cx from "classnames";
+import {
+  type HTMLAttributes,
+  type TableHTMLAttributes,
+  forwardRef,
+} from "react";
 
 import EntityItem from "metabase/components/EntityItem";
 import IconButtonWrapper from "metabase/components/IconButtonWrapper";
 import Link from "metabase/core/components/Link";
 import AdminS from "metabase/css/admin.module.css";
-import type { TextProps } from "metabase/ui";
+import type { IconProps, TextProps } from "metabase/ui";
 import { FixedSizeIcon, Text } from "metabase/ui";
 
 import { RawMaybeLink } from "../Badge/Badge.styled";
 
 import type { ResponsiveProps } from "./utils";
 import { getContainerQuery } from "./utils";
 
-export const Table = styled.table<{ isInDragLayer?: boolean }>`
+type TableProps = TableHTMLAttributes<HTMLTableElement> & {
+  isInDragLayer?: boolean;
+};
+
+export const Table = styled((props: TableProps) => (
+  <table {...props} className={cx(props.className, AdminS.ContentTable)} />
```

Comment:
I updated this to use `cx` as styled components merges the incoming `className` with the `className` of the styles in the styled components when using `styled.table`, so the existing `defaultProps.className` is actually merged